src/css/style.css

```diff
@@ -0,0 +1,83 @@
+h1{
+	padding-top: 10px;
+	color: black;
+	text-align: center;
+	font-size: 40px;
+	font-family: verdana;
+	font-weight: bold;
+
+}
+
+button{
+	position: absolute;
+	top: 65;
+	right: 210;
+
+}
+
+#timelinePlaceholder{
+	width: 1500px;
+	border-style: solid;
+	border-width: 1px;
+	margin: auto;
+}
+
+div.tweet {
+	padding-top: 10px;
+	padding-bottom: 10px;
+	height: 110px;
+	width: 1500px;
+	border-style: solid;
+	border-bottom-width: 1px;
+	border-top-width: 0px;
+	border-right-width: 0px;
+	border-left-width: 0px;
+	border-color: lightgrey;
+}
+
+img{
+	height: 60;
+	width: 60;
+	border-radius: 50%;
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+
+}
+
+div.userName{
+	padding-top: 5px;
+	text-align: center;
+	font-weight: bold;
+	font-size: 15px;
+}
+
+div.twitterHandle{
+	text-align: center;
+	font-size: 12px;
+}
+
+div.dateBlock{
+	font-size: 12px;
+	padding-bottom: 5px;
+	font-style: italic;
+}
+
+div.leftColumn{
+	padding-top: 10px;
+	padding-left: 10px;
+	height: 100px;
+	width: 10%;
+	float: left;
```

Comment:
float works.  You will need to use `clear: both` in the proper area if you go with floats